ARGO/src/GameScreen.cpp

```diff
@@ -192,13 +207,16 @@ void GameScreen::updatePlayers(float t_deltaTime)
 {
 	for (Entity& player : m_players)
 	{
-		m_inputSystem.update(player);
-		m_commandSystem.update(player, m_eventManager);
-		m_aiSystem.update(player);
-		m_healthSystem.update(player);
-		m_transformSystem.update(player, t_deltaTime);
-		m_collisionSystem.update(player);
-		m_particleSystem.update(player, t_deltaTime);
+		if (static_cast<HealthComponent*>(player.getComponent(ComponentType::Health))->isAlive())
+		{
+			m_inputSystem.update(player);
+			m_commandSystem.update(player, m_eventManager);
```

Comment:
This is fine now because you are going straight to menu when players die, but you might want to keep input system/command system outside of this if check, if we do have a proper game over/win screen